src/index.js

```diff
@@ -38,28 +42,82 @@ toggleBtn.addEventListener('click', function () {
 
 //display data on page////
  function displayData(data, word) {
-    console.log(data);
-    infoWord.innerText = `${word}`;
-    //display phonetic symbol
-    // const phonetic = data[0].phonetic
-    // phoneticPara.innerHTML = phonetic;
-     //display meanings
-    const meanings = data[0].meanings[0].definitions;
-    let meaningList = ``;
-    meanings.forEach((meaning, ind) => {
-      meaningList += ` ${ind + 1}. ${meaning.definition}  \n  sentence: ${meaning.example} \n synonyms: ${meaning.synonym } \n`;
-      
-      definition.innerText = meaningList
-      });
-    //display examples
-    let exampleList = ``;
-    meanings.forEach((meaning) => {
-        exampleList += `${meaning.example} \n`;
-        example.innerText = exampleList
-        });
+     if ( data.result) {  // if api can't find word
+         error.classList.remove('hidden')
+     } else {
+        error.classList.add('hidden')
+         console.log(data);
+         infoWord.innerText = `${word}`;
+         phoneticPara.innerText =`${data[0].phonetic}`
+         
+          //play phonetics audio
+         let getPhoneticAudio = data[0].phonetics.find((item, index) => {
+             if (item.audio.length > 0) return true
+           })
+           audio.setAttribute('src', getPhoneticAudio.audio)
+           function playAudio() {
+             audio.play()
+           }
+           audioBtn.addEventListener('click', playAudio)
+         
+          //display meanings
+          const meanings = data[0].meanings;
+          let meaningList = ''
+           meanings.map((item)=> {
+               meaningList += displayMeaings(item)
+           })
```

Comment:
This made me remember that you have not joined when I gave the talk about legal issues in software development.

When you lift a substantial amount of code from a source, always acknowledge the source in a comment block. Code plagiarism is a thing. 😒